### PR TITLE
Clean controllers

### PR DESCRIPTION
**Pull request by nunopolonia**
- Change Controller.Errors to ErrorController
- Change actions to either send error atoms or tuples with error atoms and http error code
- Add action_fallback to controllers
- Extract building of the info_start and info_end maps from the voting controller to the vote action
- Remove unused sync endpoint for invites in the admin area

### CODE REVIEW

**Comment by nunopolonia** [comment]
This PR is based on the remove projects PR instead of master since it didn't make sense to make this changes to the projects since they were being removed. It should only be merged after that PR

**Comment by nunopolonia** [comment]
[Clean controllers in the backend using  phoenix 1.3's action_fallback](https://trello.com/c/61bqvch4/119-clean-controllers-in-the-backend-using-phoenix-13s-actionfallback)

**Comment by nunopolonia** [comment]
[Add dangling commas to the backend](https://trello.com/c/6Rs8Rphl/117-add-dangling-commas-to-the-backend)

File: lib/api_web/controllers/error_controller.ex [hunk 1]
```diff
@@ -1,48 +1,53 @@
-defmodule ApiWeb.Controller.Errors do
+defmodule ApiWeb.ErrorController do
   use Api.Web, :controller
 
   alias ApiWeb.{ChangesetView, ErrorView}
 
-  def changeset(conn, data) do
+  def call(conn, {:error, changeset}) do
```
Comment (pkoch) [inline lib/api_web/controllers/error_controller.ex]: Let's force the type on matching. Something like `{:error, %Ecto.Changeset{} = changeset}`.

**Comment by nunopolonia** [comment]
Done, :shipit: ?

**Comment by pkoch** [comment]
:shipit:!

**Pull request merged**